pkg/cli/table.go

```diff
@@ -7,7 +7,7 @@ import (
 
 type Table table.Table
 
-var defaultTableWidth = 120
+var defaultTableWidth = 180
```

Comment:
We can detect the size of the terminal with https://cs.opensource.google/go/x/term/+/v0.19.0:term.go;l=51

Reply:
Updated with breakpoints.